diff -c pg_prometheus/src/prom.c pg_prometheus/src/prom.c
*** pg_prometheus/src/prom.c	2018-08-17 16:04:20.000000000 +0300
--- pg_prometheus/src/prom.c	2019-03-24 10:40:30.450637000 +0200
***************
*** 404,410 ****
  	TimestampTz ts = PG_GETARG_TIMESTAMPTZ(0);
  	text	   *name = PG_GETARG_TEXT_PP(1);
  	float8		value = PG_GETARG_FLOAT8(2);
! 	Jsonb	   *jb = PG_GETARG_JSONB(3);
  
  	char	   *metric_name = text_to_cstring(name);
  	PrometheusJsonbParseCtx ctx = {0};
--- 404,414 ----
  	TimestampTz ts = PG_GETARG_TIMESTAMPTZ(0);
  	text	   *name = PG_GETARG_TEXT_PP(1);
  	float8		value = PG_GETARG_FLOAT8(2);
!         #ifdef PG_GETARG_JSONB_P
! 	    Jsonb   *jb = PG_GETARG_JSONB_P(3);
!         #else
!             Jsonb   *jb = PG_GETARG_JSONB(3);
!         #endif
  
  	char	   *metric_name = text_to_cstring(name);
  	PrometheusJsonbParseCtx ctx = {0};
